File: WorldFeed/Server/src/WorldFeed.Streams/Model/AccountActivity/ActivityStreamUserRevokedAppPermissionsDTO.cs
namespace WorldFeed.Streams.Model.AccountActivity
{
    using System;
    using Newtonsoft.Json;

    public class ActivityStreamUserRevokedAppPermissionsDTO
    {
        [JsonProperty("date_time")]
        public DateTime DateTime { get; set; }

        [JsonProperty("target")]
        public ActivityStreamAppIdentifierDTO Target { get; set; }

        [JsonProperty("source")]
        public ActivityStreamUserIdentifierDTO Source { get; set; }
    }
}

File: WorldFeed/Client/src/app/streams/Model/AccountActivity/ActivityStreamUserRevokedAppPermissionsDTO.ts
import {ActivityStreamAppIdentifierDTO} from "./ActivityStreamAppIdentifierDTO";
import {ActivityStreamUserIdentifierDTO} from "./ActivityStreamUserIdentifierDTO";

export class ActivityStreamUserRevokedAppPermissionsDTO {
  // [JsonProperty("date_time")]
  public DateTime: DateTimeOffset;

  // [JsonProperty("target")]
  public Target: ActivityStreamAppIdentifierDTO;

  // [JsonProperty("source")]
  public Source: ActivityStreamUserIdentifierDTO;
}
